tic_tac_toe/game.py

```diff
@@ -24,16 +26,26 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
-
+    if (not isinstance(position, tuple) 
+        or not (isinstance(position[0], int) and isinstance(position[1], int))
+        or not len(position) == 2
+        or not ((0 <= position[0] < 3) and (0 <= position[1] < 3))):
+            
```

Comment:
With the way I ended up doing it, I kinda feel like line 30 is unnecessary since line 32 serves the same purpose?
Line 32 here checks if w.e. you have at position[0] and [1] are integers between 0 and 2, so this step right here is also making sure that it is an instance of an integer.

booleans also simplify down to False = 0, and True = 1, so i think parameters like (True, False) would pass the test, but you don't want those.

nitpicky, but you could also do 0 <= x <= 2, instead of 0 <= x < 3, i think it makes it slightly more readable